lib/blue_apron/spree_client.rb

```diff
@@ -133,6 +133,16 @@ def get_products(options = {})
       response
     end
 
+    def get_homepage_products
+      response = get "/api/products/homepage"
+      response.keys.each do |key|
+        response[key].each do |product|
+          sanitize_product!(product)
+        end
+      end
+      response
```

Comment:
non-blocking: this hash iteration could be simplified a bit
```ruby
response.each do |_key, product|
  sanitize_product!(product)
end
```

Reply:
That doesn't quite work - each key in the response isn't a product, but rather a list of products. But the following would work, I believe:

```
response.each do |_key, products|
  products.each { |product| sanitize_product!(product) }
end
```

Thanks for the suggestion - much cleaner this way!

Reply:
Definitely fine as is, but I think you can also do
```ruby 
response.values.flatten.each do |product|
  sanitize_product!(product)
end
```